tst/EnergyPlus/api/TestDataTransfer.c

```diff
@@ -61,7 +61,7 @@ void afterZoneTimeStepHandler(EnergyPlusState state)
 {
     printf("STARTING A NEW TIME STEP\n");
     if (handlesRetrieved == 0) {
-        if (!apiDataFullyReady(state)) return;
+        if (apiDataFullyReady(state) == 1) return;
```

Comment:
Small fix.  The function returns 0 for true (success), so need to check if it's non zero.  The == 1 is technically redundant because 1 is truthy, but it doesn't hurt anything to be explicit here.